lib/scout_apm/instruments/http.rb

```diff
@@ -0,0 +1,46 @@
+module ScoutApm
+  module Instruments
+    class HTTP
+      attr_reader :context
+
+      def initialize(context)
+        @context = context
+        @installed = false
+      end
+
+      def logger
+        context.logger
+      end
+
+      def installed?
+        @installed
+      end
+
+      def install
+        if defined?(::HTTP) && defined?(::HTTP::Client)
+          @installed = true
+
+          logger.info "Instrumenting HTTP"
```

Comment:
Change to "Instrumenting HTTP::Client" since we already have 2 other HTTP instruments ([http_client](https://github.com/scoutapp/scout_apm_ruby/blob/master/lib/scout_apm/instruments/http_client.rb) and [net_http](https://github.com/scoutapp/scout_apm_ruby/blob/master/lib/scout_apm/instruments/net_http.rb))